frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.html

```diff
@@ -69,6 +69,17 @@ <h5 class="fw-light">Challenge Settings</h5>
           (click)="stopParticipation($event)"
           ><strong class="text-light-black fs-15">Close Participation</strong>
         </mat-checkbox>
+        <div class="ev-action">
+          <div *ngIf="isChallengeHost" class="col-sm-1.5 col-xs-1.5 ev-upload-action">
+            <span class="fs-16 fw-light">
+              <strong class="text-light-black">Overview: </strong>
```

Comment:
Can we please make it `Challenge overview` instead of `Overview:` ?

Reply:
I think we are moving all the edit details to the new card https://github.com/Cloud-CV/EvalAI/pull/3451#issuecomment-858337248 so I will add the `overview` in the `Edit Description` card once that PR is approved

Reply:
Since, we are not adding the new section, I will move the `Challenge overview` just under the challenge title.